src/styles/_utilities-deprecated.scss

```diff
@@ -0,0 +1,13 @@
+@forward "media/mixins";
+@forward "media/variables";
+@forward "media/placeholders";
+@forward "theme/placeholders";
+@forward "theme/variables-deprecated";
+@forward "typography/mixins";
+@forward "typography/placeholders";
+@forward "components/button/placeholders";
+
+$_one-rem-in-pixel: 16;
+@function pixel-to-rem($pixel-size) {
+  @return calc($pixel-size / $_one-rem-in-pixel) * 1rem;
+}
```

Comment:
Le fichier `_utilities.scss` existe encore avec le même contenu (à la différence de l'import `@forward "theme/variables-deprecated";` évidemment)

Est-ce qu'on ne voudrait pas avoir : 
- un fichier `_utilities-deprecated.scss` qui ne contient que le `@forward "theme/variables-deprecated";` 
- et un fichier `_utilities.scss` qui continue d'exposer tout le reste?

Reply:
J'ai créé un fichier `utilities.scss` qui correspond à l'utilities migré vers l'UI kit et l'ancien fichier a été renommé en `utilities-deprecated` pour garder le legacy :)

Reply:
Ah je pensais ne mettre dans les `_utilities-deprecated.scss` que ce qui est remplacé par une version plus à jour / en phase avec l'UI Kit

(Sinon là j'ai par exemple une function `pixel-to-rem` qui provient de `_utilities-deprecated.scss` alors qu'elle n'est pas dépréciée)

Reply:
C'est une question qu'on s'est posé hier avec Gauthier, Su et Fred. Si on fait ce que tu proposes, on va devoir repasser sur toutes les pages pour gérer la distinction entre les couleurs et les autres propriétés, ce qu'on a pas forcement envie de faire